e2e/uimodel/agent/agent.go

```diff
@@ -3,20 +3,23 @@ package agent
 import (
 	"fmt"
 
-	utils "github.com/gravitational/robotest/e2e/model/ui"
+	"github.com/gravitational/robotest/e2e/uimodel/utils"
 	"github.com/gravitational/robotest/infra"
 
 	. "github.com/onsi/gomega"
 	web "github.com/sclevine/agouti"
 	. "github.com/sclevine/agouti/matchers"
 )
 
+// AgentServer is agent server ui model
 type AgentServer struct {
+	// Hostname is agent server hostname
 	Hostname    string
 	cssSelector string
 	page        *web.Page
 }
 
+// CreateAgentServer finds agent servers on the page and creates ui models
 func CreateAgentServer(page *web.Page, cssSelector string) AgentServer {
 	elem := page.Find(cssSelector)
 	Expect(elem).To(BeFound())
```

Comment:
As I understand it'll panic with very minimal messaging about what happened if the element is not there. At least maybe make is say the URL of page and CSS element name.